django/__main__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+"""
+Alias to allow the django module to be run as a script.
+
+Example:
+
+    python -m django collectstatic
+"""
+from django.core import management
+
+if __name__ == "__main__":
```

Comment:
Interestingly, no. If this module is imported from another package, it's name will be `django.__main__`. You could use that to, for example, allow you to have `django.__main__:management.execute_from_command_line` in the setup.py entry points without it executing it twice. (not that we'd want to do that here, but the pattern makes sense in other contexts)